tools/pipelines/templates/build-npm-package.yml

```diff
@@ -181,9 +181,15 @@ stages:
         # Setup
         - checkout: self
           clean: true
-          lfs: ${{ parameters.checkoutSubmodules }}
+          lfs: true
           submodules: ${{ parameters.checkoutSubmodules }}
 
+        # Initialize Git LFS to ensure test assets have been downloaded
+        - script: |
+            git lfs fetch
+            git lfs pull
+          displayName: git-lfs
```

Comment:
Not sure if it's just the syntax highlighting in Github but this line looks like it's being interpreted as part of the script. I suggest we make the taks  a full "Bash@3" task just for clarity.

Reply:
Modified